test/integration/default/files/_mapdata/amazonlinux-1.yaml

```
@@ -0,0 +1,34 @@
+# yamllint disable rule:indentation rule:line-length
+# Amazon Linux AMI-2018
+---
+added_in_defaults: defaults_value
+added_in_lookup: lookup_value
+added_in_pillar: pillar_value
+arch: amd64
+config: /etc/template-formula.conf
+lookup:
+  added_in_lookup: lookup_value
+  master: template-master
+  winner: lookup
+master: template-master
+pkg:
+  name: bash
+rootgroup: root
+service:
+  name: systemd-journald
```

Once the [new PR](https://github.com/saltstack-formulas/template-formula/pull/203#discussion_r481297085) is merged, this will become:

```suggestion
pkg:
  name: cronie
rootgroup: root
service:
  name: crond
```

---

Fix provided: #204.